models/addresses/ethereum/addresses_ethereum_optimism_batchinbox_combinations.sql

```diff
@@ -0,0 +1,37 @@
+{{ config(alias = 'optimism_batchinbox_combinations',
```

Comment:
can you add `schema` in all config blocks?

Reply:
on it